toolkit/components/uniffi-js/UniFFIScaffolding.cpp

```diff
@@ -80,7 +80,7 @@ already_AddRefed<Promise> UniFFIScaffolding::CallAsync(
   }
 #endif
 
-  aError.ThrowUnknownError(nsPrintfCString("Unknown function id: %ld", aId));
```

Comment:
The compiler was giving me warnings about those so I modified them while I was passing through